infra/helper.py

```diff
@@ -71,6 +71,9 @@
 LANGUAGES_WITH_BUILDER_IMAGES = {'go', 'jvm', 'python', 'rust', 'swift'}
 ARM_BUILDER_NAME = 'oss-fuzz-buildx-builder'
 
+COV_BASE_URL = 'https://storage.googleapis.com/'
+COV_BACKUP_URL = '-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/'
```

Comment:
nit: `CORPUS_BACKUP_URL`?

Reply:
Good idea: I changed the name to `HTTPS_CORPUS_BACKUP_URL_FORMAT ` in order to resemble the existing corpus GCS bucket URL https://github.com/google/oss-fuzz/blob/fa0e49f4b40f2ddb5d227d9247977e7d7ecb7283/infra/helper.py#L59-L64